pkg/handler/loki.go

```diff
@@ -199,8 +199,8 @@ func processLineFilters(key string, values []string, lineFilters *strings.Builde
 		if i > 0 {
 			regexStr.WriteByte('|')
 		}
-		if strings.Contains(value, "`") {
-			return errors.New("backquote not authorized in flows requests")
+		if strings.Contains(value, "`") || strings.Contains(value, "&") {
+			return errors.New("unauthorized sign in flows requests")
```

Comment:
Should we put a regex like `^[\w-]+$` in both front & back to be safe ?

This will match any alphanumeric & underscore + dash characters. I'm not sure we need more for our fields

Reply:
Maybe `.` dots for ips and `:` double dots if allowed in pods / namespace ?

Reply:
I am not a fan of regexp, but I guess this is fine here, I will replace by a regexp!